### DISCUSSION TITLE
Problem with Adapter SONOFF Zigbee 3.0 USB Dongle Plus

### DISCUSSION THREAD

**Discussion by B3n84** [Koenkk/zigbee2mqtt#10574] [General]
Hi all,
I have a problem with the Sonoff Adaper (TI CC2652P + CP2102N) and Zigbee2MQTT. The problem is that the Docker-container with Zigbee2Mqtt constantly restarts . In the logs I read that there is a problem with zigbee-herdsman.

My System:
SONOFF Zigbee 3.0 USB Dongle Plus
Rpi 4, 2GB Ram

The software Zigbee2MQTT, Mosquitto and Node-Red are running in docker containers (docker-compose). Everything worked for a day or two and I was able to control a Philips Hue lightbulb. But it happened a few times that this problem occurred after restarting the Raspberry Pi and the official power supply 0(5V / 3A).

So far I have tried the following without success:
- Firmware flashed: CC1352P2_CC2652P_launchpad_coordinator_20211217 
- Sonoff Dongel in every USB-Port of the Pi
- test -w [PORT] && echo success || echo failure =success

dmesg -w: 
`[ 3049.631267] veth6ed5f35: renamed from eth0
[ 3049.762097] IPv6: ADDRCONF(NETDEV_CHANGE): veth6ed5f35: link becomes ready
[ 3049.762235] br-38b64f8e3047: port 3(veth1ff5058) entered blocking state
[ 3049.762242] br-38b64f8e3047: port 3(veth1ff5058) entered forwarding state
[ 3049.904057] br-38b64f8e3047: port 3(veth1ff5058) entered disabled state
[ 3049.920805] device veth1ff5058 left promiscuous mode
[ 3049.920817] br-38b64f8e3047: port 3(veth1ff5058) entered disabled state
[ 3050.094290] usb 1-1.3: new full-speed USB device number 4 using xhci_hcd
[ 3050.251916] usb 1-1.3: New USB device found, idVendor=10c4, idProduct=ea60, bcdDevice= 1.00
[ 3050.251936] usb 1-1.3: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[ 3050.251952] usb 1-1.3: Product: Sonoff Zigbee 3.0 USB Dongle Plus
[ 3050.251967] usb 1-1.3: Manufacturer: ITead
[ 3050.251983] usb 1-1.3: SerialNumber: 3a6041d3d5c9eb118259914f1d69213e
[ 3050.254688] cp210x 1-1.3:1.0: cp210x converter detected
[ 3050.265146] usb 1-1.3: cp210x converter now attached to ttyUSB0
[ 3109.490135] br-38b64f8e3047: port 3(veth43964b9) entered blocking state
[ 3109.490156] br-38b64f8e3047: port 3(veth43964b9) entered disabled state
[ 3109.490585] device veth43964b9 entered promiscuous mode
[ 3109.592602] IPv6: ADDRCONF(NETDEV_CHANGE): veth89516cb: link becomes ready
[ 3109.592865] IPv6: ADDRCONF(NETDEV_CHANGE): veth43964b9: link becomes ready
[ 3109.592938] br-38b64f8e3047: port 3(veth43964b9) entered blocking state
[ 3109.592945] br-38b64f8e3047: port 3(veth43964b9) entered forwarding state
[ 3112.355487] br-38b64f8e3047: port 3(veth43964b9) entered disabled state
[ 3112.356894] eth0: renamed from veth89516cb`

Log File:
`debug 2022-01-04 21:54:38: Loaded state from file /app/data/state.json
info  2022-01-04 21:54:38: Logging to console and directory: 'data/log/' filename: log.txt
info  2022-01-04 21:54:38: Starting Zigbee2MQTT version 1.22.2 (commit #414c51f)
info  2022-01-04 21:54:38: Starting zigbee-herdsman (0.13.188)
debug 2022-01-04 21:54:38: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null,"delay":null,"disableLED":false},"backupPath":"/app/data/coordinator_backup.json","databaseBackupPath":"/app/data/database.db.backup","databa>error 2022-01-04 21:54:39: Error while starting zigbee-herdsman
debug 2022-01-04 21:54:54: Loaded state from file /app/data/state.json
info  2022-01-04 21:54:54: Logging to console and directory: 'data/log/' filename: log.txt`

Maybe someone has a hint or advice what I can try to solve the problem.

Thanks in advance,
Ben

**Comment by B3n84** [comment]
Zigbee2MQTT worked properly for a few days. After restarting the RaspberryPI (no changes on Zigbee2MQTT), the same problem occurs again. Is it possible that Zigbee2Mqtt is having a problem with USB devices? After a few restarts Zigbee2MQTT will work until the Pi is restarted.

**Comment by Sazelish** [comment]
I'm having the same issue's. The dongle is a couple of weeks old. Before installing Zigbee2Mqtt I have been using Zigbee Home Automation without any issues. Yesterday I have installed an configure Zigbee2Mqtt and my Pi keeps on rebooting without any warnings. It happens almost every 2 to 4 minutes. I'm not able to retrieve any errormessage from the logfiles


Sonoff_Zigbee_3.0_USB_Dongle_Plus
RpI 4
